python/invoke_release/tasks.py

```diff
@@ -1589,31 +1594,33 @@ def wheel(_):
     ))
 
 
-def open_pull_request(base, head, title, token):
-    remote = subprocess.check_output(
-        ['git', 'remote', 'get-url', 'origin'],
-        stderr=subprocess.STDOUT,
+def open_pull_request(branch_name, current_branch_name, version_to_release, github_token):
+    remote = six.text_type(
+        subprocess.check_output(
+            ['git', 'remote', 'get-url', 'origin'],
+            stderr=subprocess.STDOUT,
         )
+    )
     repo = (remote.split(':')[1].split('.')[0])
     url = 'https://api.github.com/repos/{}/pulls'.format(repo)
 
     values = {
-      'title': title,
-      'base': base,
-      'head': head
+        'title': 'Released version {}'.format(version_to_release),
+        'base': current_branch_name,
+        'head': branch_name,
     }
 
     body = json.dumps(values).encode('utf-8')
     headers = {
         'Content-Type': 'application/json',
-        'Authorization': 'token {}'.format(token),
+        'Authorization': 'token {}'.format(github_token),
         'Accept': 'application/vnd.github.v3+json',
-        'Content-Length': len(body)
+        'Content-Length': len(body),
     }
 
     try:
         req = urllib.request.Request(url, body, headers)
         with closing(urllib.request.urlopen(req)) as f:
```

Comment:
the way it is implemented seems to indicate it is simpler to just call `f.close()` in a `finally:` block instead of using `closing`